### PR TITLE
Develop-celine

### PR DESCRIPTION
**Pull request by celinejacques**

### CODE REVIEW

File: data_splitting.py [hunk 1]
```diff
@@ -121,8 +122,10 @@ def process_data_splitting_scenario(scenario):
         else:
             raise NameError('This testset_option [' + scenario.testset_option + '] scenario is not recognized')
             
-        node = Node(x_node_train, x_node_test, y_node_train, y_node_test)
+        node = Node(x_node_train, x_node_test, y_node_train, y_node_test, 
+                    str(nodeId))
```
Comment (bowni) [inline data_splitting.py]: Are you sure @celinejacques that it has to be stringified here? Looks like in the rest of the code you keep it an integer and stringify it each time you print it.

File: data_splitting.py [hunk 2]
```diff
@@ -105,6 +105,7 @@ def process_data_splitting_scenario(scenario):
   
     # Populate nodes
     node_list = []
+    nodeId=1
```
Comment (bowni) [inline data_splitting.py]: @celinejacques could you reformat it to `node_id` to enforce the snake_case style?

File: fl_training.py [hunk 3]
```diff
@@ -138,8 +138,8 @@ def compute_test_score(node_list):
                           verbose=0,
                           validation_data=(node.x_val, node.y_val))
                 
-                val_acc_list.append(history.history['val_acc'])
-                acc_list.append(history.history['acc'])
+                val_acc_list.append(history.history['val_accuracy'])
+                acc_list.append(history.history['accuracy'])
```
Comment (bowni) [inline fl_training.py]: For my culture, is this a Keras usage best practice?

File: data_splitting.py (hunk 1 above)
Comment (celinejacques) [reply data_splitting.py]: My idea was that the name will be a string such as the name of the center. 
Which option do you prefer: 
- I change node number to string before initialize it (so nodeId or node_id now is a string and not stringify it here)
- I left it as a number and stringify it only for print. 
- Other idea? 
I will change the prints when it is needed with the option you prefer.

File: fl_training.py (hunk 3 above)
Comment (celinejacques) [reply fl_training.py]: Sorry in the first version that I used to test 'acc' did not work. I needed to write 'accuracy' and 'val_accuracy' (same for Samya). But I had to install again everything and now it works with 'acc'.

**Comment by bowni** [comment]
Thanks @celinejacques , pretty cool! @RomainGoussault I let you merge? As you are currently developing on a number of issues, I think it is best that you merge yourself so that you can then locally pull and rebase your current work.

File: data_splitting.py [hunk 4]
```diff
@@ -105,6 +105,7 @@ def process_data_splitting_scenario(scenario):
   
     # Populate nodes
     node_list = []
+    node_id=1
```
Comment (RomainGoussault) [inline data_splitting.py]: We should start at 0.

Comment (celinejacques) [reply data_splitting.py]: @RomainGoussault No problem, it is done

**Comment by RomainGoussault** [comment]
Thanks @celinejacques !

**Pull request merged**